Validation/RecoEgamma/plugins/PhotonValidatorMiniAOD.cc

```diff
@@ -0,0 +1,376 @@
+  histname = "hcalTowerSumEtConeDR04";
+  h_hcalTowerSumEtConeDR04_[0][0] = iBooker.book1D(histname+"All_miniAOD",   "hcalTowerSumEtConeDR04: All Ecal",etBin,etMin,20.);
+  h_hcalTowerSumEtConeDR04_[0][1] = iBooker.book1D(histname+"Barrel_miniAOD","hcalTowerSumEtConeDR04: Barrel ", etBin,etMin,20.);
+  h_hcalTowerSumEtConeDR04_[0][2] = iBooker.book1D(histname+"Endcap_miniAOD","hcalTowerSumEtConeDR04: Endcap ", etBin,etMin,20.);
+  //
+  histname = "hcalTowerBcSumEtConeDR04";
+  h_hcalTowerBcSumEtConeDR04_[0][0] = iBooker.book1D(histname+"All_miniAOD",   "hcalTowerBcSumEtConeDR04: All Ecal",etBin,etMin,20.);
+  h_hcalTowerBcSumEtConeDR04_[0][1] = iBooker.book1D(histname+"Barrel_miniAOD","hcalTowerBcSumEtConeDR04: Barrel ", etBin,etMin,20.);
+  h_hcalTowerBcSumEtConeDR04_[0][2] = iBooker.book1D(histname+"Endcap_miniAOD","hcalTowerBcSumEtConeDR04: Endcap ", etBin,etMin,20.);
+  histname = "isoTrkSolidConeDR04";
+  h_isoTrkSolidConeDR04_[0][0] = iBooker.book1D(histname+"All_miniAOD",   "isoTrkSolidConeDR04: All Ecal",etBin,etMin,etMax*0.1);
+  h_isoTrkSolidConeDR04_[0][1] = iBooker.book1D(histname+"Barrel_miniAOD","isoTrkSolidConeDR04: Barrel ", etBin,etMin,etMax*0.1);
+  h_isoTrkSolidConeDR04_[0][2] = iBooker.book1D(histname+"Endcap_miniAOD","isoTrkSolidConeDR04: Endcap ", etBin,etMin,etMax*0.1);
+  histname = "nTrkSolidConeDR04";
+  h_nTrkSolidConeDR04_[0][0] = iBooker.book1D(histname+"All_miniAOD",   "nTrkSolidConeDR04: All Ecal",20,0., 20) ;
+  h_nTrkSolidConeDR04_[0][1] = iBooker.book1D(histname+"Barrel_miniAOD","nTrkSolidConeDR04: Barrel ", 20,0., 20) ;
+  h_nTrkSolidConeDR04_[0][2] = iBooker.book1D(histname+"Endcap_miniAOD","nTrkSolidConeDR04: Endcap ", 20,0., 20) ;
+
+
+  //  Infos from Particle Flow - isolation and ID
+  histname = "chargedHadIso";
+  h_chHadIso_[0]=  iBooker.book1D(histname+"All_miniAOD",   "PF chargedHadIso:  All Ecal",etBin,etMin,20.);
+  h_chHadIso_[1]=  iBooker.book1D(histname+"Barrel_miniAOD",   "PF chargedHadIso:  Barrel",etBin,etMin,20.);
+  h_chHadIso_[2]=  iBooker.book1D(histname+"Endcap_miniAOD",   "PF chargedHadIso:  Endcap",etBin,etMin,20.);
+  histname = "neutralHadIso";
+  h_nHadIso_[0]=  iBooker.book1D(histname+"All_miniAOD",   "PF neutralHadIso:  All Ecal",etBin,etMin,20.);
+  h_nHadIso_[1]=  iBooker.book1D(histname+"Barrel_miniAOD",   "PF neutralHadIso:  Barrel",etBin,etMin,20.);
+  h_nHadIso_[2]=  iBooker.book1D(histname+"Endcap_miniAOD",   "PF neutralHadIso:  Endcap",etBin,etMin,20.);
+  histname = "photonIso";
+  h_phoIso_[0]=  iBooker.book1D(histname+"All_miniAOD",   "PF photonIso:  All Ecal",etBin,etMin,20.);
+  h_phoIso_[1]=  iBooker.book1D(histname+"Barrel_miniAOD",   "PF photonIso:  Barrel",etBin,etMin,20.);
+  h_phoIso_[2]=  iBooker.book1D(histname+"Endcap_miniAOD",   "PF photonIso:  Endcap",etBin,etMin,20.);
+
+
+
+}
+
+void  PhotonValidatorMiniAOD::endRun (edm::Run const& r, edm::EventSetup const & theEventSetup) {}
```

Comment:
I beg your pardon ??
I need these histograms. I am not sure what you're asking here.

 N.

______________________________________

Nancy Marinelli

Research Associate Professor
University of Notre Dame, IN, US

CERN, Bdg 40/3-A01, 1211 Geneva,

SWITZERLAND

Phone +41-22-76-70809, fax +41-22-76-78940

From: Federico De Guio <notifications@github.com<mailto:notifications@github.com>>
Reply-To: cms-sw/cmssw <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tue, 20 Oct 2015 06:51:49 -0700
To: cms-sw/cmssw <cmssw@noreply.github.com<mailto:cmssw@noreply.github.com>>
Cc: Nancy Marinelli <nancy.marinelli@cern.ch<mailto:nancy.marinelli@cern.ch>>
Subject: Re: [cmssw] Mini aodvalidation: V0 (#11750)


In Validation/RecoEgamma/plugins/PhotonValidatorMiniAOD.cc<https://github.com/cms-sw/cmssw/pull/11750#discussion_r42497132>:

> +  h_chHadIso_[1]=  iBooker.book1D(histname+"Barrel_miniAOD",   "PF chargedHadIso:  Barrel",etBin,etMin,20.);
> +  h_chHadIso_[2]=  iBooker.book1D(histname+"Endcap_miniAOD",   "PF chargedHadIso:  Endcap",etBin,etMin,20.);
> +  histname = "neutralHadIso";
> +  h_nHadIso_[0]=  iBooker.book1D(histname+"All_miniAOD",   "PF neutralHadIso:  All Ecal",etBin,etMin,20.);
> +  h_nHadIso_[1]=  iBooker.book1D(histname+"Barrel_miniAOD",   "PF neutralHadIso:  Barrel",etBin,etMin,20.);
> +  h_nHadIso_[2]=  iBooker.book1D(histname+"Endcap_miniAOD",   "PF neutralHadIso:  Endcap",etBin,etMin,20.);
> +  histname = "photonIso";
> +  h_phoIso_[0]=  iBooker.book1D(histname+"All_miniAOD",   "PF photonIso:  All Ecal",etBin,etMin,20.);
> +  h_phoIso_[1]=  iBooker.book1D(histname+"Barrel_miniAOD",   "PF photonIso:  Barrel",etBin,etMin,20.);
> +  h_phoIso_[2]=  iBooker.book1D(histname+"Endcap_miniAOD",   "PF photonIso:  Endcap",etBin,etMin,20.);
> +
> +
> +
> +}
> +
> +void  PhotonValidatorMiniAOD::endRun (edm::Run const& r, edm::EventSetup const & theEventSetup) {}


please remove

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11750/files#r42497132>.